hackathon_site/registration/models.py

```diff
@@ -56,12 +56,30 @@ class Application(models.Model):
         ("other", "Other"),
     ]
 
+    TSHIRT_SIZE_CHOICES = [
+        ("small", "S"),
+        ("medium", "M"),
+        ("large", "L"),
+        ("extra-large", "XL"),
+    ]
+
+    HACKATHON_NUMBER_CHOICES = [
+        ("one", "1"),
+        ("two", "2"),
+        ("three", "3"),
+        ("four", "4"),
+        ("five_more", "5 or more"),
+    ]
+
     user = models.OneToOneField(User, on_delete=models.CASCADE, null=False)
     team = models.ForeignKey(
         Team, related_name="applications", on_delete=models.CASCADE, null=False
     )
 
     # User Submitted Fields
+    tshirt_size = models.CharField(
+        max_length=50, choices=TSHIRT_SIZE_CHOICES, null=False
+    )
     birthday = models.DateField(null=False)
     gender = models.CharField(max_length=50, choices=GENDER_CHOICES, null=False)
     ethnicity = models.CharField(max_length=50, choices=ETHNICITY_CHOICES, null=False)
```

Comment:
I can't comment on the exact line, but the ticket says we don't need to collect address anymore. You should remove all of those except the country field, which in the template you can put under the "Personal Information" section to balance out the columns